operator/pkg/helmreconciler/reconciler.go

```diff
@@ -101,9 +101,6 @@ func NewHelmReconciler(client client.Client, restConfig *rest.Config, iop *value
 		iop = &valuesv1alpha1.IstioOperator{}
 		iop.Spec = &v1alpha1.IstioOperatorSpec{}
 	}
-	if operatorRevision, found := os.LookupEnv("REVISION"); found {
-		iop.Spec.Revision = operatorRevision
-	}
```

Comment:
Does canary upgrade with in-cluster operator https://istio.io/latest/docs/setup/install/operator/#canary-upgrade still work if we remove this?

Reply:
Yes, but the workflow is modified slightly: https://github.com/istio/istio.io/pull/8299